ktor-http/common/src/io/ktor/http/ContentTypes.kt

```diff
@@ -211,6 +211,78 @@ public class ContentType private constructor(
+        public val WordX: ContentType =
+            ContentType("application", "vnd.openxmlformats-officedocument.wordprocessingml.document")
+        public val ApiJson: ContentType = ContentType("application", "vnd.api+json")
+        public val Sh: ContentType = ContentType("application", "x-sh")
+        public val Rtf: ContentType = ContentType("application", "rtf")
+        public val Kml: ContentType = ContentType("application", "vnd.google-earth.kml+xml")
+        public val PowerPoint: ContentType = ContentType("application", "vnd.ms-powerpoint")
+        public val Tar: ContentType = ContentType("application", "x-tar")
+        public val Odt: ContentType = ContentType("application", "vnd.oasis.opendocument.text")
+        public val Ods: ContentType = ContentType("application", "vnd.oasis.opendocument.spreadsheet")
+        public val Odp: ContentType = ContentType("application", "vnd.oasis.opendocument.presentation")
+        public val Jpeg: ContentType = ContentType("application", "jpeg")
+        public val Png: ContentType = ContentType("application", "png")
+        public val Svg: ContentType = ContentType("application", "svg+xml")
+        public val Tiff: ContentType = ContentType("application", "tiff")
+        public val Webp: ContentType = ContentType("application", "webp")
+        public val H264: ContentType = ContentType("application", "x-h264")
+        public val Mpeg: ContentType = ContentType("application", "mpeg")
+        public val Ogg: ContentType = ContentType("application", "ogg")
+        public val Wav: ContentType = ContentType("application", "wav")
+        public val Mp3: ContentType = ContentType("application", "mp3")
+        public val Midi: ContentType = ContentType("application", "x-midi")
+        public val Flac: ContentType = ContentType("application", "x-flac")
+        public val Mp4: ContentType = ContentType("application", "mp4")
+        public val Mkv: ContentType = ContentType("application", "x-matroska")
+        public val Avi: ContentType = ContentType("application", "x-msvideo")
+        public val Iso: ContentType = ContentType("application", "x-iso9660-image")
+        public val Bzip2: ContentType = ContentType("application", "x-bzip2")
+        public val TarGz: ContentType = ContentType("application", "x-tar+gzip")
+        public val Xlsm: ContentType = ContentType("application", "vnd.ms-excel.sheet.macroenabled.12")
+        public val Mobi: ContentType = ContentType("application", "x-mobipocket-ebook")
+        public val Epub: ContentType = ContentType("application", "epub+zip")
+        public val JsonLd: ContentType = ContentType("application", "ld+json")
+        public val Xul: ContentType = ContentType("application", "xul+xml")
+        public val VndMsPowerPoint: ContentType = ContentType("application", "vnd.ms-powerpoint")
+        public val VndMsWord: ContentType = ContentType("application", "msword")
```

Comment:
I can't see KTOR-7108, but IMHO many items in this list are strange. Eg VndMsWord: the vnd prefix is not used in the content type and there is already a word content type defined in line 215.
Or what is VndPlain? It's not in the list: https://www.iana.org/assignments/media-types/media-types.xhtml#application